### ISSUE TITLE
Fix false positives for jQuery CSS function in value-keyword-case

### ISSUE THREAD

**Issue by lambert-velir** [stylelint/stylelint#4919]
```javascript
$(".class").css({
  width: 100 * (slick.options.slidesToShow / slick.slideCount) + "%",
});
```

In this example in a javascript file, stylelint is changing `slidesToShow` to `slidestoshow`

### Which rule, if any, is the bug related to?

value-keyword-case

### What stylelint configuration is needed to reproduce the bug?

e.g.

```json
{
  "rules": {
    "value-keyword-case": "lower",
  }
}
```

### Which version of stylelint are you using?

13.3.3

### How are you running stylelint: CLI, PostCSS plugin, Node.js API?

stylelint command line and vscode extension (atom extension does not seem to lint js files)

### Does the bug relate to non-standard syntax (e.g. SCSS, Less etc.)?

no

I'm not sure what the best solution is for this.   Can stylelint be smart enough to only apply this rule to strings in this case of an object of styles?

**Comment by jeddy3** [comment]
@lambert-velir Thanks for the report.

I can reproduce this [using the demo](https://stylelint.io/demo#N4Igxg9gJgpiBcIAkAKAOiAdGANgQwGcCMBKbIlYNAOwAJaB3ASygBcALeWgRgAZfaAKlooCOJmADWmCAAdWTCNQKYxLGAQAqEAMrsIDWgHpaaqavGwAwhACu1ViVoBqWhgCkGADQ0AviQBuEC8QAgBPBzwADwRwIgBaJmp4gCsCYPAlADMmAHNYqjo3EAAnWxwNDC5C+noMADc8HFsYeMkYMIYIEqh4sEIYKuKcAxgSjBp6Xz8QXyA).

It's likely a problem with [the parser](https://github.com/stylelint/postcss-css-in-js) stylelint uses [to support CSS-in-JS](https://stylelint.io/about/syntaxes). I've added this issue to https://github.com/stylelint/stylelint/issues/4574, which is a growing list of the problems with the syntax that we need help to fix.

Please consider [contributing to the parser](https://stylelint.io/developer-guide/syntaxes) if you have time.

**Comment by jeddy3** [comment]
Closing [as stylistic rules are frozen](https://github.com/stylelint/stylelint/issues/5674). The community is welcome to migrate the rule [to a plugin](https://stylelint.io/developer-guide/plugins) and fix this bug.
